tests/extensions/test_nested_min_max.py

```diff
@@ -0,0 +1,93 @@
+# Licensed under the GPL: https://www.gnu.org/licenses/old-licenses/gpl-2.0.html
```

Comment:
These tests are unnecessary if we use the functional test framework. Let's remove this file.